### DISCUSSION TITLE
Hello, I'm trying to build de project using maven but it doesn't work

### DISCUSSION THREAD

**Discussion by githubdevcode** [eclipse-ee4j/angus-mail#103] [General]
Hi everyone, I've downloaded angus mail project and when I try to build de proyect using maven, maven throws the following error: 

Failed to execute goal on project [36mangus-core[m: [1;31mCould not resolve dependencies for project org.eclipse.angus:angus-core:jar:2.0.2-SNAPSHOT: jakarta.mail:jakarta.mail-api:jar:2.1.2 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced[m -> [1m[Help 1][m
[[1;31mERROR[m] 

Please, I need help with this error, thanks

**Comment by githubdevcode** [comment]
I need to extend the class SMTPTransport to create a new protocol of my own. I need to generate angus-mail jar file with the new changes implemented with my new modifications that I required. Please, help me @lukasj

**Comment by lukasj** [comment]
use `-Pstaging` till the API is published in central (it's ready, I only need to get a green light for release from others)
